libgloss/arc/iotdk.specs

```diff
@@ -3,7 +3,7 @@
 %rename startfile iotdk_startfile
 
 *link:
-%(iotdk_link) -marcv2elfx --defsym=ivtbase_addr=0x20000000
+%(iotdk_link)
```

Comment:
Like in the example which I've gave to you, but you don't need to define `ivtbase_addr`